fixup-mountpoints

```diff
@@ -115,6 +115,36 @@ case "$DEVICE" in
             -e 's block/platform/msm_sdcc.1/by-name/LTALabel mmcblk1 ' \
             "$@"
         ;;
+    "z3c" | "d5803" | "aries")
```

Comment:
We can omit the 'aries' and 'd5803' for now, the CM12 based build has codename 'z3c' so that will work.